src/main/java/net/fabricmc/loom/configuration/providers/minecraft/mapped/IntermediaryMinecraftProvider.java

```diff
@@ -45,6 +45,7 @@ public IntermediaryMinecraftProvider(Project project, M minecraftProvider) {
 
 	@Override
 	public final MappingsNamespace getTargetNamespace() {
+		if (extension.isNeoForge()) return MappingsNamespace.MOJANG;
```

Comment:
I guess this making this provider a no-op? A short comment here would be good too